doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -463,6 +463,10 @@ Modem libraries
 
 * Removed the deprecated A-GPS library.
 
+* Fixed:
+
+  * NCSIDB-713 where the compiler would generate an error when building :file:`ncs/nrf/include/modem/lte_lc.h` with C++ applications.
```

Comment:
I'd rather not refer to the Jira issue. If the reader is not one of us (Nordicans), they will not have a clue what the issue was. Maybe something like:

An issue where the compiler would generate...

Reply:
oh but we have done so earlier. See https://github.com/nrfconnect/sdk-nrf/blob/5813de2c78645d6436c479ed68973914882b7939/doc/nrf/releases/release-notes-changelog.rst#L738 for example

Reply:
I am okay with removing the jira task ID. I thought it is a convention to add the JIRA task ID when I looked at earlier similar additions.

Reply:
Fixed and pushed

Reply:
Thanks Balaji! I know we still have a few Jira references in our release notes, but we haven't managed to make a clear decision on this matter. This is purely my personal opinion to the use of them. I hope this will be decided soon.

Reply:
It seems we have now agreed to still use the Jira refs. But I think this entry works well without it.